### PR TITLE
PacBio: Initial addition of `isoseq3`

### PR DESCRIPTION
**Pull request by pb-dseifert**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/isoseq3/0.7.2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+{% set version = "0.7.2" %}
+{% set sha256 = "bd3e272dddb6153bd949e70cb57718c92c52608bc76ced8e95a08c5ee94b52b0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: isoseq3
+  url: https://github.com/PacificBiosciences/IsoSeq3/releases/download/v{{ version }}/isoseq3 # [linux]
```
Comment (bgruening) [inline recipes/isoseq3/0.7.2/meta.yaml]: @pb-dseifert is it possible to compile isoseq3 from source?

File: recipes/isoseq3/0.7.2/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  sha256: {{ sha256 }} # [linux]
+
+about:
+  home: https://github.com/PacificBiosciences/IsoSeq3
+  license: Pacific Biosciences Proprietary
+  summary: IsoSeq3 - Scalable De Novo Isoform Discovery
+
+extra:
```
Comment (bgruening) [inline recipes/isoseq3/0.7.2/meta.yaml]: Any DOI what we can add?

Comment (pb-dseifert) [reply recipes/isoseq3/0.7.2/meta.yaml]: nope, nothing published

File: recipes/isoseq3/0.7.2/meta.yaml (hunk 1 above)
Comment (pb-dseifert) [reply recipes/isoseq3/0.7.2/meta.yaml]: IsoSeq is one of those applications that PacBio does not want to open-source at this stage.

**Pull request merged**